### DISCUSSION TITLE
Sequel Ace Triggers Definer

### DISCUSSION THREAD

**Discussion by cedric-jacquot** [Sequel-Ace/Sequel-Ace#1401] [Ideas]
Hello !

I can't update Sequel Ace user (definer) triggers. I need to run specific SQL query.

Do you have a solution for this ? Or can you create a new feature for this ?

Thanks,

Cédric J.

**Comment by Jason-Morcos** [comment]
Generally, to report bugs please open a new issue instead of a discussion. When submitting the issue, please fill out the full issue template including particularly the version of Sequel Ace being used, version of MySQL/MariaDB, exact error message/behavior being observed, and any logs that may be applicable.

We try to be as helpful as possible, but the volume of issues reported limits us to only have time to dig into those that include all the details needed. I encourage you to open a new issue with all the details provided.
